exercises/zipper/package.yaml

```diff
@@ -1,12 +1,13 @@
 name: zipper
-version: 1.1.0.4
+version: 1.1.0.6
```

Comment:
you somehow increased this version by two... which I guess is not a completely terrible thing to do, but which I suppose we should avoid if we can.

Reply:
Oops!